src/blssignature.cpp

```diff
@@ -60,6 +60,73 @@ void BLSInsecureSignature::GetPoint(relic::ep2_st* output) const {
     *output = *sig;
 }
 
+bool BLSInsecureSignature::Verify(const uint8_t* msg, size_t len, const BLSPublicKey& pubKey) const {
```

Comment:
I'm not super convinced on BLSInsecureSignature having such a different interface compared to BLSSignature